src/pages/Home.jsx

```diff
@@ -11,7 +11,10 @@ import SponsorsSection from '@/layouts/SponsorsSection'
 
 function Home() {
   return (
-    <main>
+    <main aria-labelledby="main-heading">
```

Comment:
This element was removed [here](https://github.com/GDG-Detroit/devfest-website/pull/45/files)

The accompanying change was made in PageLayout, which this will now use